tests/foreman/ui/test_gpgkey.py

```diff
@@ -1159,9 +1157,8 @@ def test_key_associate_14(self):
 
     @run_only_on('sat')
     def test_key_associate_15(self):
-        """@test: Create gpg key with valid name and valid gpg key
-        then associate it with empty (no repos) custom
-        product then delete it
+        """@test: Create gpg key with valid name and valid gpg key then
```

Comment:
just to let you know - here `@test` is not mandatory.  Testimony considers the first line as the test name now.